stumpy/floss.py

```diff
@@ -476,6 +498,15 @@ def __init__(
         self._custom_iac = custom_iac
         self._normalize = normalize
         self._p = p
```

Comment:
I think it's important to initialize some variables:

```
self._T_subseq_isconstant = None
self._M_T = None
self._Σ_T = None
```

Reply:
Fixed!